cdap-master/pom.xml

```diff
@@ -491,6 +491,9 @@
                         <exclude name="${additional.artifacts.exclude.pattern}"/>
                         <exclude name="**/target/*-sources.jar"/>
                         <exclude name="**/target/*-javadoc.jar"/>
+                        <!-- Wrangler excludes --> 
+                        <exclude name="**/target/wrangler-core*"/>
+                        <exclude name="**/target/wrangler-test*"/>
                         <!--We don't want to package the following plugins with CDAP-->
                         <exclude name="**/target/solrsearch-plugins*"/>
```

Comment:
Ali that is right. Long term, each plugin will be in its own repo. Then cdap-build repo will contain what plugins to bring in.